README.md

```diff
@@ -6,7 +6,91 @@ For more details read the [TSC Charter](https://github.com/nodejs/TSC/blob/maste
+* https://github.com/nodejs/help — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/http — administration delegated to the HTTP Working Group
+* https://github.com/nodejs/http2 — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/iojs.org — administration delegated to the Website Working Group
+* https://github.com/nodejs/installer — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/Intl — administration delegated to the Intl Working Group
+* https://github.com/nodejs/llnode — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/nan — administration delegated to the Addon API Working Group
+* https://github.com/nodejs/node-addon-examples — administration delegated to the Addon API WorkingGroup)
+* https://github.com/nodejs/node-chakracore — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/node-gyp — administration delegated to the Core Technical Committee
+* https://github.com/nodejs/nodereport — administration delegated to the Post Mortem Working Group
+* https://github.com/nodejs/post-mortem — administration delegated to the Post Mortem Working Group
+* https://github.com/nodejs/promises — administration delegated to the Promises Working Group
+* https://github.com/nodejs/readable-stream — administration delegated to the Streams Working Group
+* https://github.com/nodejs/summit
+* https://github.com/nodejs/testing — administration delegated to the Testing Working Group
+* _And_ various language and resource translation groups under https://github.com/nodejs/
+
+### Responsibilities
+
+The TSC exercises autonomy in setting up and maintaining procedures, policies, and management and administrative structures as it deems appropriate for the maintenance and operation of these projects and resources.
+
+Included in the responsibilities of the TSC are:
+
+* Managing code and documentation creation and changes for the listed projects and resources
+* Setting and maintaining standards covering contributions of code, documentation and other materials
+* Managing code and binary releases: types, schedules, frequency, delivery mechanisms
+* Making decisions regarding dependencies of the Node.js Core project, including what those dependencies are and how they are bundled with source code and releases
+* Creating new repositories and projects under the _nodejs_ GitHub organization as required
+* Setting overall technical direction for the Node.js Core project, including high-level goals and low-level specifics regarding features and functionality
+* Setting and maintaining appropriate standards for community discourse via the various mediums under TSC control
+* Setting and maintaining governance rules for the conduct and make-up of the TSC, CTC, Working Groups and other bodies under the TSC
```

Comment:
To avoid the Top Down view of the Org- we could try to avoid the word "under"... instead use "within the TSC's domain" or similar.